### PR TITLE
Update _config.yaml

### PR DESCRIPTION
**Pull request by rachwo**
adding dark version of header - don't know if it will work but is an attempt

### CODE REVIEW

File: _config.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: MATCH Lab
 subtitle: Michael Smith Laboratories, UBC
 description: Multiomic Analysis of Treatment Resistance and Cancer Heterogeneity
-header: images/MATCH_logo.png
+header: images/MATCH_logo_light.svg
 footer: images/background.jpg
-header-dark: false
+header-dark: images/MATCH_logo_dark.svg
```
Comment (ytakemon) [inline _config.yaml]: `header-dark` needs to be a true/false statement. Please switch back to `false`

File: _config.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@
 title: MATCH Lab
 subtitle: Michael Smith Laboratories, UBC
 description: Multiomic Analysis of Treatment Resistance and Cancer Heterogeneity
-header: images/MATCH_logo.png
+header: images/MATCH_logo_light.svg
```
Comment (ytakemon) [inline _config.yaml:5]: Approved. We can try this transparent background, but below needs fixing

**Review by ytakemon — CHANGES_REQUESTED** [review]
Please make changes.

**Comment by ytakemon** [comment]
FYI - typically, you can preview your changes to the website using the automatic git actions `🚀 Deployed preview to... `. This will appear shortly after you create your pull request.

**Comment by rachwo** [comment]
oops, thanks for your patience as i test things!

**Comment by rachwo** [comment]
i believe i've made the requested revisions (on rachwo-patch-2). is this correct? (learning how to make and modify pull requests)

**Review by ytakemon — APPROVED** [review]
Looks good lets give it a go

**Comment by rachwo** [comment]
alas, it did not work. i will pick my battles.

**Pull request merged**